ert3/console/__init__.py

```diff
@@ -3,6 +3,7 @@
 from pathlib import Path
```

Comment:
For the same reasons as below, let us try to do `import pathlib`

Reply:
I don't see the reason for having this as a completely hard rule. Using `from package import function` is pretty normal behaviour. There's nothing in PEP or similar that suggest we _shouldn't_ do this. Could you please elaborate on why you want to be so strict?

Reply:
I am also curious. I have never seen this argument before. Why shouldn't these imports follow the law of Demeter?